### PR TITLE
Dtspo 6035 dynatrace upgrade aat

### PR DESCRIPTION
**Pull request by nickmoyni**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-6035

### Change description ###

Dynatrace upgrade AAT environment adding in to dynatrace namespace.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/monitoring/aat/base/kustomization.yaml [hunk 1]
```diff
@@ -8,8 +8,6 @@ resources:
   - prometheus-values.yaml
   - monitoring-values.yaml
   - botkube-values.yaml
-  - oneagent-values.yaml
   - list-all-pods-clusterrole.yaml
 
-patchesStrategicMerge:
-  - ../../dynatrace/aat/oneagent.yaml
\ No newline at end of file
+patchesStrategicMerge:
```
Comment (hannah38) [inline apps/monitoring/aat/base/kustomization.yaml:13]: Can you fix this, the floating empty patches section (can you check the perftest one as well please)

File: apps/dynatrace/aat/00/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (nickmoyni) [reply apps/dynatrace/aat/00/kustomization.yaml:1]: These have been added for  future activities so if anyone wishes to carryout a task the structure is already in place for them to quickly commence their taks.

File: apps/monitoring/aat/base/kustomization.yaml (hunk 1 above)
Comment (nickmoyni) [reply apps/monitoring/aat/base/kustomization.yaml]: Removed.

File: apps/monitoring/aat/base/kustomization.yaml [hunk 3]
```diff
@@ -8,8 +8,6 @@ resources:
   - prometheus-values.yaml
   - monitoring-values.yaml
   - botkube-values.yaml
-  - oneagent-values.yaml
```
[the remark this answers is not in the document]
Comment (nickmoyni) [reply apps/monitoring/aat/base/kustomization.yaml:11]: My change is removing this, this is the original mines is on the right its stripping out oneaget, and the strategic patch.

[the remark this answers is not in the document]
Comment (hannah38) [reply apps/monitoring/aat/base/kustomization.yaml:11]: Can you check this full list in your editor even, if you check it you should still see dynatrace deployed in monitoring

[the remark this answers is not in the document]
Comment (hannah38) [reply apps/monitoring/aat/base/kustomization.yaml:11]: and if you use https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#testing-in-local to test it locally on monitoring namespace, should still see dynatrace resources

**Review by hannah38 — CHANGES_REQUESTED** [review]

File: apps/monitoring/aat/base/kustomization.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (nickmoyni) [reply apps/monitoring/aat/base/kustomization.yaml:11]: dynatrace resource for AAT removed from monitoring list.

**Review by hannah38 — APPROVED** [review]

**Pull request merged**